pyaedt/modeler/cad/Primitives.py

```diff
@@ -275,45 +587,4908 @@ def version(self):
+            vArg2.append("DuplicateMirrorNormalZ:="), vArg2.append(Znorm)
+            vArg3 = ["NAME:Options", "DuplicateAssignments:=", duplicate_assignment]
+            if is_3d_comp:
+                orig_3d = [i for i in self.user_defined_component_names]
+            added_objs = self.oeditor.DuplicateMirror(vArg1, vArg2, vArg3)
+            self.add_new_objects()
+            if is_3d_comp:
+                added_3d_comps = [i for i in self.user_defined_component_names if i not in orig_3d]
+                if added_3d_comps:
+                    self.logger.info("Found 3D Components Duplication")
+                    return added_3d_comps
+            return added_objs
+        else:
+            vArg1 = ["NAME:Selections", "Selections:=", selections, "NewPartsModelFlag:=", "Model"]
+            vArg2 = ["NAME:MirrorParameters"]
+            vArg2.append("MirrorBaseX:="), vArg2.append(Xpos)
+            vArg2.append("MirrorBaseY:="), vArg2.append(Ypos)
+            vArg2.append("MirrorBaseZ:="), vArg2.append(Zpos)
+            vArg2.append("MirrorNormalX:="), vArg2.append(Xnorm)
+            vArg2.append("MirrorNormalY:="), vArg2.append(Ynorm)
+            vArg2.append("MirrorNormalZ:="), vArg2.append(Znorm)
+
+            self.oeditor.Mirror(vArg1, vArg2)
+            return True
+
+    @pyaedt_function_handler()
+    def move(self, objid, vector):
+        """Move objects from a list.
+
+        Parameters
+        ----------
+        objid : list, Position object
+            List of object IDs.
+        vector : list
+            Vector of the direction move. It can be a list of the ``[x, y, z]``
+            coordinates or a Position object.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+
+        References
+        ----------
+
+        >>> oEditor.Move
+        """
+        Xvec, Yvec, Zvec = self._pos_with_arg(vector)
+        szSelections = self.convert_to_selections(objid)
+
+        vArg1 = ["NAME:Selections", "Selections:=", szSelections, "NewPartsModelFlag:=", "Model"]
+        vArg2 = ["NAME:TranslateParameters"]
+        vArg2.append("TranslateVectorX:="), vArg2.append(Xvec)
+        vArg2.append("TranslateVectorY:="), vArg2.append(Yvec)
+        vArg2.append("TranslateVectorZ:="), vArg2.append(Zvec)
+
+        if self.oeditor is not None:
+            self.oeditor.Move(vArg1, vArg2)
+        return True
+
+    @pyaedt_function_handler()
+    def duplicate_around_axis(
+        self,
+        objid,
+        cs_axis,
+        angle=90,
+        nclones=2,
+        create_new_objects=True,
+        is_3d_comp=False,
+        duplicate_assignment=True,
+    ):
+        """Duplicate a selection around an axis.
+
+        Parameters
+        ----------
+        objid : list, str, int, Object3d or UserDefinedComponent
+            Name or ID of the object.
+        cs_axis :
+            Coordinate system axis or the Application.AXIS object.
```

Comment:
```suggestion
            Coordinate system axis or the ``Application.AXIS`` object.
```
Please add the one or more data types.